e2e/libs/snapshot/base.py

```diff
@@ -0,0 +1,72 @@
+from abc import ABC, abstractmethod
+from utility.utility import set_annotation
+from utility.utility import get_annotation_value
+
+class Base(ABC):
+
+    ANNOT_ID = "test.longhorn.io/snapshot-id"
```

Comment:
Should move to the [constant.py](https://github.com/longhorn/longhorn-tests/blob/297533fc8894a203f1777b304c25c2f2119771c6/e2e/libs/utility/constant.py#L7-L8).

Reply:
Refined. Thank you!

Reply:
Looks like this is still here. Will this be moved to a constant file?

Reply:
Refined. Thank you!